cmd/createenr.go

```diff
@@ -61,22 +63,36 @@ func runCreateEnrCmd(w io.Writer, config p2p.Config, dataDir string) error {
 		return err
 	}
 
-	localEnode, db, err := p2p.NewLocalEnode(config, key)
+	enrStr, err := createENR(key, config)
 	if err != nil {
-		return errors.Wrap(err, "failed to open enode")
+		return err
 	}
-	defer db.Close()
 
 	keyPath := p2p.KeyPath(dataDir)
 
 	_, _ = fmt.Fprintf(w, "Created ENR private key: %s\n", keyPath)
-	_, _ = fmt.Fprintln(w, localEnode.Node().String())
+	_, _ = fmt.Fprintln(w, enrStr)
 
 	writeEnrWarning(w, keyPath)
 
 	return nil
 }
 
+func createENR(key *ecdsa.PrivateKey, config p2p.Config) (string, error) {
```

Comment:
add a comment since it isn't obvious why we do stuff twice

Reply:
sure